schemas/constants.yml

```diff
@@ -132,25 +132,33 @@ dependency-relation:
 
 # Description of actions. They are defined on the provisioner
 action-description: |
-  Actions are defined on the provisioner. When an action is triggered,
-  it will use the `url` and `context` property to construct the URL and
-  perform a `POST` request. Note that the request will be signed with the
-  user's Taskcluster credentials.
+  Actions are defined on the provisioner and they are performed via a `POST` request using the `url`.
+  Depending on the `context`, the `url` will be able to fill path parameters. In addition, `context` is
+  used by the front-end to know where to display the action. For example, `context=worker`
+  will display the action on the worker explorer.
+
+  _Note: The request will be signed with the user's Taskcluster credentials._
 
 # Description of the action context
 action-context-description: |
   Actions have a "context" that is one of provisioner, worker-type,
-  or worker, indicating which it applies to. `context` is used to construct
-  the query string of the `POST` request.
-  If `context='worker'`, the query string will be
-  `?provisionerId=${PROVISIONER_ID}&workerType=${WORKER_TYPE}&workerGroup=${WORKER_GROUP}&workerId=${WORKER_ID}`.
-  If `context='worker-type'`, the query string will be
-  `?provisionerId=${PROVISIONER_ID}&workerType=${WORKER_TYPE}`.
-  If `context='provisioner'`, the query string will be
-  `?provisionerId=${PROVISIONER_ID}`.
+  or worker, indicating which it applies to. `context` is used by the front-end to know where to display the action.
+
+  | `context`   | Path parameters                                      | Page displayed        |
+  |-------------|------------------------------------------------------|-----------------------|
+  | provisioner | :provisionerId                                       | Worker explorer       |
```

Comment:
Provisioner Explorer

Reply:
Also, you can probably omit the path parameters here, as they are included in action-url-description.